targets/LuaSdk/make.js

```diff
@@ -198,7 +202,7 @@ function getResultAction(tabbing, apiCall) {
 }
 
 function getAuthentication(apiCall) {
-    if (apiCall.url === "/Authentication/GetEntityToken")
+    if (apiCall.url === "/Authentication/GetEntityToken" || apiCall.url === "/GameServerIdentity/AuthenticateGameServerWithCustomId")
```

Comment:
Similar to above, AuthenticateGameServerWithCustomId can be treated as a normal EntityToken auth method for this purpose.